### PR TITLE
Answers oak story added

### PR DESCRIPTION
**Pull request by spjika**

### CODE REVIEW

**Comment by mapmeld** [comment]
This is a great adoption and community reuse of the Honolulu Answers platform! GitHubbers please include this story

**Comment by jlord** [comment]
Haha, thanks @mapmeld! And super thanks @spjika! I love this story and it's so relevant in many ways!

Just trying to clear out some of the other pull requests and am wondering if we should pace out story additions?

@benbalter what are your thoughts? We definitely don't want stories to go cold, but should we let each story sit on the front page for at least a couple days before we add a new one? Or maybe we just add them as they come in?

@mapmeld @spjika thoughts on that yourselves as users?

**Comment by spjika** [comment]
I would agree that at least a few days live makes sense yes, too high rotation is bad. No rush!

File: _posts/2013-10-15-OakAnswers.md [hunk 1]
```diff
@@ -0,0 +1,24 @@
+---
+layout: story
+author: Spike, OpenOakland
+title: Public-Private Collaborations
+description: " Oakland Answers is a collaborative web site that allows residents easy access to common questions without th eusual pain of navigating a city website."
+image: /images/streetmix.png
```
Comment (benbalter) [inline _posts/2013-10-15-OakAnswers.md]: Believe you'll want to update the image path here.

File: _posts/2013-10-15-OakAnswers.md [hunk 2]
```diff
@@ -0,0 +1,24 @@
...
+category: open source
+---
+[Oakland Answers](http://answers.oaklandnet.com) was an opensource public private collaboration between OpenOakland, the City of Oakland and our community. 
+It is a very simple, searchable tool to find answers to common questions- it works like google, it works on your phone and it's beautiful.
+
+To build this, we gathered 70 people from across our city to build a resource for the entire city as part f the National Day of Civic Hacking.
```
Comment (benbalter) [inline _posts/2013-10-15-OakAnswers.md]: Per [the style guide](https://github.com/github/government.github.com/blob/gh-pages/CONTRIBUTING.md), wherever possible, would be awesome to talk about the project in the third person.

File: _posts/2013-10-15-OakAnswers.md [hunk 3]
```diff
@@ -0,0 +1,24 @@
+---
+layout: story
+author: Spike, OpenOakland
+title: Public-Private Collaborations
+description: " Oakland Answers is a collaborative web site that allows residents easy access to common questions without th eusual pain of navigating a city website."
+image: /images/streetmix.png
+category: open source
+---
+[Oakland Answers](http://answers.oaklandnet.com) was an opensource public private collaboration between OpenOakland, the City of Oakland and our community.
```
Comment (benbalter) [inline _posts/2013-10-15-OakAnswers.md]: > was an opensource public private collaboration

Was? Is the project no longer maintained?

File: _posts/2013-10-15-OakAnswers.md [hunk 4]
```diff
@@ -0,0 +1,24 @@
...
+description: " Oakland Answers is a collaborative web site that allows residents easy access to common questions without th eusual pain of navigating a city website."
+image: /images/streetmix.png
+category: open source
+---
+[Oakland Answers](http://answers.oaklandnet.com) was an opensource public private collaboration between OpenOakland, the City of Oakland and our community. 
+It is a very simple, searchable tool to find answers to common questions- it works like google, it works on your phone and it's beautiful.
+
+To build this, we gathered 70 people from across our city to build a resource for the entire city as part f the National Day of Civic Hacking.  
+
+We took the raw code originally built for Honolulu by [Code for America](http://www.codeforamerica.org), and forked it on Github, then tweaked it with city staff to create a new resource.
```
Comment (benbalter) [inline _posts/2013-10-15-OakAnswers.md]: Github -> GitHub :smile:

**Comment by benbalter** [comment]
This is awesome. Thanks for taking the time to put this together. Would love to feature your efforts.

It may be worthwhile to take a look at [the style guide](https://github.com/github/government.github.com/blob/gh-pages/CONTRIBUTING.md) a bit, but perhaps we could shoot to get this published for next week?

**Comment by spjika** [comment]
Ben take a look, all the issues you raised are fixed now.

**Comment by benbalter** [comment]
@spjika Amazing. Can't wait to get this added. It looks like you link out to `answers.png` but it's really `answers.jpg` (which is why CI is failing. If you're able to make that quick change, I'd say we can :shipit: 

Thanks again for taking the time to put this together, and for the edits. Such a greats story.

**Comment by spjika** [comment]
image link fixed

**Comment by benbalter** [comment]
Awesome. Let's plan on posting this Monday-ish. Thanks again.

**Comment by benbalter** [comment]
It should appear on the front-page momentarily, and can be linked to directly at http://government.github.com/stories/public-private-collaborations/.

Thanks again for taking the time to submit the pull request. Such an awesome story and glad we can help to tell it.

**Pull request merged**